runtime/bcutil/attrlookup.h

```diff
@@ -170,7 +170,9 @@ lookupKnownAttribute (register const char *str, register unsigned int len)
 #line 71 "attrlookup.gperf"
       {"NestMembers", CFR_ATTRIBUTE_NestMembers, CFR_ATTRIBUTE_NestMembers},
 #line 70 "attrlookup.gperf"
-      {"MethodParameters", CFR_ATTRIBUTE_MethodParameters, CFR_ATTRIBUTE_MethodParameters}
+      {"MethodParameters", CFR_ATTRIBUTE_MethodParameters, CFR_ATTRIBUTE_MethodParameters},
```

Comment:
did you regenerate this file with gperf?

Reply:
I have now lol